# MULTI_EVENT_IMU
Topic: "/dvs/events"
CamHeight: 180
CamWidth: 240
Fx: 199.0923665423112
Fy: 198.8288204700886
Cx: 132.1920713777002
Cy: 110.7126600112956
P_Fx: 168.6294097900391
P_Fy: 178.5641784667969
P_Cx: 135.348079770296
P_Cy: 113.6189973794753
K1: -0.3684363117977873
K2: 0.1509472435566583
P1: -0.0002961305343848646
P2: -0.000759431726241032
K3: 0.0